beacon-chain/builder/testing/mock.go

```diff
@@ -16,6 +16,7 @@ import (
 type MockBuilderService struct {
 	HasConfigured         bool
 	Payload               *v1.ExecutionPayload
+	CapellaPayload        *v1.ExecutionPayloadCapella
```

Comment:
I suggest renaming to `PayloadCapella` because we append fork names to the end everywhere else.